### PR TITLE
A11Y: Fix lighthouse check with new auth system

### PR DESCRIPTION
**Pull request by benjdlambert**
Looks like we need to run the new backend system with the `GuestProvider` installed to get through the auth check.

Also, look like the additional `page.goto(...)` in `guest-auth` was causing some issues with local storage contexts for some reason, this is still unclear as to why removing this works.

### CODE REVIEW

File: lighthouserc.js [hunk 1]
```diff
@@ -16,6 +16,7 @@
 module.exports = {
   ci: {
     collect: {
+      // headful: true,
```
Comment (emmaindal) [inline lighthouserc.js:19]: did you mean to leave this here for easier local dev or was it meant to be cleaned up after our session? 😬

**Review by emmaindal — APPROVED** [review]
💯 just one comment around the headful config!

File: lighthouserc.js (hunk 1 above)
Comment (benjdlambert) [reply lighthouserc.js:19]: Bit of both 😅 Can remove it to be honest, and we can pick it up in docs instead.

Comment (benjdlambert) [reply lighthouserc.js]: I think `autorun --headful` does the same thing anyways.

**Review by freben — APPROVED** [review]

**Pull request merged**